packages/ui-components/src/components/layout/sidebar/menu/group/Item.vue

```diff
@@ -2,13 +2,15 @@
   <component
     :is="linkComponent"
     v-if="!hasChildren"
+    v-tippy="tooltipText"
     :to="to"
-    class="group flex items-center justify-between space-x-2 shrink-0 text-body-xs select-none rounded-md w-full hover:bg-primary-muted py-1 px-5 cursor-pointer"
+    class="group flex items-center space-x-2 shrink-0 text-body-xs text-foreground select-none rounded-md w-full py-1 px-5"
+    :class="[!disabled && 'cursor-pointer hover:bg-primary-muted']"
     exact-active-class="bg-foundation-focus hover:!bg-foundation-focus"
     :external="external"
     :target="external ? '_blank' : undefined"
   >
-    <div class="flex items-center space-x-2">
+    <div class="flex items-center space-x-2" :class="[disabled && 'opacity-60']">
```

Comment:
Using opacity here because we don't really have a good color grey for this, they are either too dark or too light